### PR TITLE
User feedback docs for dotnet

### PR DESCRIPTION
**Pull request by Tyrrrz**
cc @bruno-garcia

### CODE REVIEW

File: src/platforms/apple/common/enriching-events/user-feedback.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Sentry
 
 let eventId = SentrySDK.capture(message: "My message.")
 
-let userFeedback = UserFeedack(eventId: eventId)
+let userFeedback = UserFeedback(eventId: eventId)
```
Comment (bruno-garcia) [inline src/platforms/apple/common/enriching-events/user-feedback.mdx:16]: The name is actually wrong on the `SWIFT_NAME`: https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/Public/SentryUserFeedback.h#L11

Comment (bruno-garcia) [reply src/platforms/apple/common/enriching-events/user-feedback.mdx:16]: Code needs fixing first.

```suggestion
let userFeedback = UserFeedack(eventId: eventId)
```

File: src/platforms/apple/common/enriching-events/user-feedback.mdx [hunk 2]
```diff
@@ -31,3 +31,27 @@ userFeedback.email = @"john.doe@example.com";
...
+var eventId = SentrySdk.CaptureMessage("My message.");
+
+var userFeedback = new UserFeedback(
+  eventId,
+  "john.doe@example.com",
+  "It broke.",
+  "John Doe"
+);
+
+SentrySdk.CaptureUserFeedback(userFeedback);
```
Comment (bruno-garcia) [inline src/platforms/apple/common/enriching-events/user-feedback.mdx:47]: There's an extension method that takes the individual arguments, did I get it right?

Does it compile?

```suggestion
SentrySdk.CaptureUserFeedback(eventId,
  "john.doe@example.com",
  "It broke.",
  "John Doe");
```

File: src/platforms/apple/common/enriching-events/user-feedback.mdx (hunk 1 above)
Comment (bruno-garcia) [reply src/platforms/apple/common/enriching-events/user-feedback.mdx:16]: https://github.com/getsentry/sentry-cocoa/pull/829/

**Comment by Tyrrrz** [comment]
@bruno-garcia yep. updated

**Review by bruno-garcia — APPROVED** [review]

**Comment by bruno-garcia** [comment]
@Tyrrrz seems like it needs some more settings:

This looks right, because the user feedback in this case is from the Front end:

https://sentry-docs-git-user-feedback-dotnet.sentry.dev/platforms/dotnet/guides/aspnet/enriching-events/user-feedback/

But here: 
https://sentry-docs-git-user-feedback-dotnet.sentry.dev/platforms/dotnet/enriching-events/user-feedback/

Better to show either the C# version or a clarification that there are two.

And the other guides don't show user feedback

https://sentry-docs-git-user-feedback-dotnet.sentry.dev/platforms/dotnet/guides/serilog/

**Comment by Tyrrrz** [comment]
The first two are actually the same page, I'm not sure how to separate them from each other.

**Comment by bruno-garcia** [comment]
@PeloWriter could you help us out? Do you know how we can do this?

**Comment by bruno-garcia** [comment]
I fixed this on another PR #2747

**Pull request closed without merging**